src/components/TypoReport/style.js

```diff
@@ -0,0 +1,17 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  appBar: {
```

Comment:
add please padding: "10px" and we need to fix the main layout because this bar overlaps with content.